lib/rubocop/cop/style/commented_keyword.rb

```diff
@@ -59,6 +59,7 @@ class CommentedKeyword < Base
 
         SUBCLASS_DEFINITION = /\A\s*class\s+(\w|::)+\s*<\s*(\w|::)+/.freeze
         METHOD_DEFINITION = /\A\s*def\s/.freeze
+        END_DEFINITION = /\A\s*end/.freeze
```

Comment:
Wouldn't it be preferable to use a single regular expression with a name like `METHOD_OR_END_DEFINITIONS` or an official name for the `#:` syntax in RBS::Inline? I'm not sure what it's called.

```suggestion
        METHOD_OR_END_DEFINITIONS = /\A\s*(def\s|end)/.freeze
```

Reply:
I chose to use `METHOD_OR_END_DEFINITIONS` because according to [the official docs](https://github.com/soutaro/rbs-inline/wiki/Syntax-guide#-syntax), the `#:` syntax is not named, just called `#:` syntax. So I don't know its name.

Thanks for your suggestion!!